repository/feature_flag_repo.go

```diff
@@ -88,10 +88,16 @@ func (f FeatureFlagRepo) Add(ctx context.Context, config ...domain.FlagConfig) e
 	return nil
 }
 
-// Remove removes all feature entries for given environment id
-func (f FeatureFlagRepo) Remove(ctx context.Context, id string) error {
+// Remove removes the feature entry from the cache
+func (f FeatureFlagRepo) Remove(ctx context.Context, identifier string) error {
+	fcKey := domain.NewFeatureConfigsKey(identifier)
```

Comment:
Should this be `domain.NewFeatureConfigKey`?

Reply:
good spot